src/modelframe.jl

```diff
@@ -177,7 +177,7 @@ Extract the response column, if present.  `DataVector` or
 """
 function StatsBase.model_response(mf::ModelFrame)
     if mf.terms.response
-        convert(Array, mf.df[mf.terms.eterms[1]])
+        Missings.disallowmissing(convert(Array, mf.df[mf.terms.eterms[1]]))
```

Comment:
Since there's already a call to `convert` (and provided that it cannot be removed, i.e. that we really need an `Array`), better do `convert(Array{Missings.T(eltype(mf.df[mf.terms.eterms[1]])}, mf.df[mf.terms.eterms[1]])` (or something prettier using a temporary variable). That will avoid making an intermediate copy in some cases.